app/pages/landing.js

```diff
@@ -0,0 +1,209 @@
+  color: '#003366',
+  marginBottom: '15px',
+  fontWeight: 'bold'
+};
+
+class Landing extends Component {
+  state = { show: false };
+
+  showModal = e => {
+    e.preventDefault();
+    this.setState({ show: true });
+    console.log(this.state);
+  };
+
+  hideModal = e => {
+    this.setState({ show: false });
+  };
+
+  renderModal = () => {
+    if (this.state.show === true) {
+      return (
+        <Modal
+          centered
+          size="lg"
+          show={this.state.show}
+          onHide={this.hideModal}
+          data-testid="modal"
+        >
+          <Modal.Header closeButton>
+            <Modal.Title style={{ color: '#003366' }}>
+              You will now be redirected to the BCEID site
+            </Modal.Title>
+          </Modal.Header>
+          <Modal.Body style={{ padding: '25px' }}>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat. <p /> Duis aute irure dolor in
+            reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+            pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+            culpa qui officia deserunt mollit anim id est laborum.
+          </Modal.Body>
+          <Modal.Footer>
+            <Button
+              style={{ backgroundColor: '#003366', float: 'left' }}
+              size="lg"
+              href="https://www.bceid.ca/"
+              target="_blank"
+              data-testid="btn-get-portal"
+              className="btn-portal"
+            >
+              Continue to BCEID registration
+            </Button>
+          </Modal.Footer>
+        </Modal>
+      );
+    }
+  };
+
+  render() {
+    return (
+      <div>
+        <Header
+          isLoggedIn
+          userName={'Mary Jane'}
+          occupation={'Admin Rio Tinto'}
+        />
+        <div style={headerBackground}>
+          <h1 style={heading}>Welcome to your CIIP Portal</h1>
+          <p style={headingSubtext}>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat.
+          </p>
+        </div>
+        {this.renderModal()}
+        <Container>
+          <Row>
+            <Col md>
+              <Card style={Cards}>
+                <Card.Body style={CardBody1}>
+                  <Card.Title style={CardTitle1}>Login with BCEID</Card.Title>
+                  <Card.Text style={{ color: 'white' }}>
+                    Applying for CIIP requires having a valid BCEID. If you have
+                    already acquired yours click below to Login
+                  </Card.Text>
+                  <Button
```

Comment:
This button should link to the login page, even if it's not implemented right now